### ISSUE TITLE
Insert and delete step buttons do not work (Trac #101)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#105]
v0.1-515-gbbde5d1

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/101
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "v0.1-515-gbbde5d1", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Insert and delete step buttons do not work", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-04-17T19:13:31", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120417T19:38:34: ryan commented:

> This error only shows up with the dmf_control_board plugin enabled.
> 
> We can also get rid of the delete button (already available with the delete key and from the right-click menu) and the insert button (add it to the right-click menu.

**Comment by cfobel** [comment]
Trac update at 20120417T23:20:20: 
* christian@fobel.net commented:

> commit 0d0c1074306e6033934f4632bf5801d8cdcadf55
> Author: Christian Fobel <christian@fobel.net>
> Date:   Tue Apr 17 19:11:27 2012 -0400
> 
>     Move insert/delete from buttons to popup menu
>     
>     Fixes #105.  Note that this also fixes #103 and fixes #104 after updating the
>     dmf_control_board plugin to revision v0.1-216-g03278e8.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "" to "microdrop"
